src/SFA.DAS.EmployerAccounts.Web/Controllers/SearchPensionRegulatorController.cs

```diff
@@ -126,19 +142,33 @@ public async Task<ActionResult> SearchPensionRegulatorByAorn(SearchPensionRegula
         [NonAction]
```

Comment:
I know you haven't added them but what's the point of these attributes on a private method?